ci-operator/step-registry/hypershift/install/hypershift-install-commands.sh

```diff
@@ -21,17 +21,27 @@ if [ "${ENABLE_HYPERSHIFT_CERT_ROTATION_SCALE}" = "true" ]; then
   EXTRA_ARGS="${EXTRA_ARGS} --cert-rotation-scale=20m"
 fi
 
-bin/hypershift install --hypershift-image="${OPERATOR_IMAGE}" \
---oidc-storage-provider-s3-credentials=/etc/hypershift-pool-aws-credentials/credentials \
---oidc-storage-provider-s3-bucket-name=hypershift-ci-oidc \
---oidc-storage-provider-s3-region=us-east-1 \
---platform-monitoring=All \
---enable-ci-debug-output \
---private-platform=AWS \
---aws-private-creds=/etc/hypershift-pool-aws-credentials/credentials \
---aws-private-region="${HYPERSHIFT_AWS_REGION}" \
---external-dns-provider=aws \
---external-dns-credentials=/etc/hypershift-pool-aws-credentials/credentials \
---external-dns-domain-filter=service.ci.hypershift.devcluster.openshift.com \
---wait-until-available \
-${EXTRA_ARGS}
+if [ "${CLOUD_PROVIDER}" != "AZURE" ]; then
+  bin/hypershift install --hypershift-image="${OPERATOR_IMAGE}" \
+  --oidc-storage-provider-s3-credentials=/etc/hypershift-pool-aws-credentials/credentials \
+  --oidc-storage-provider-s3-bucket-name=hypershift-ci-oidc \
+  --oidc-storage-provider-s3-region=us-east-1 \
+  --platform-monitoring=All \
+  --enable-ci-debug-output \
+  --private-platform=AWS \
+  --aws-private-creds=/etc/hypershift-pool-aws-credentials/credentials \
+  --aws-private-region="${HYPERSHIFT_AWS_REGION}" \
+  --external-dns-provider=aws \
+  --external-dns-credentials=/etc/hypershift-pool-aws-credentials/credentials \
+  --external-dns-domain-filter=service.ci.hypershift.devcluster.openshift.com \
+  --wait-until-available \
+  ${EXTRA_ARGS}
+fi
+
+if [ "${CLOUD_PROVIDER}" = "AZURE" ]; then
```

Comment:
don't you need "==" here?

Reply:
can use both "=" and "==" for comparison in this case